packages/svg-to-component/lib.js

```diff
@@ -1,15 +1,18 @@
 /* eslint-disable no-param-reassign */
-const { promises: fs } = require('fs');
-const path = require('path');
+import { promises as fs } from 'fs';
+import { createRequire } from 'module';
+import { extname as _extname, basename, resolve } from 'path';
 
-const { chalk } = require('@4c/cli-core/ConsoleUtilities');
-const { transformAsync } = require('@babel/core');
-const yaml = require('js-yaml');
-const camelCase = require('lodash/camelCase');
-const upperFirst = require('lodash/upperFirst');
-const Svgo = require('svgo');
+import { chalk } from '@4c/cli-core/ConsoleUtilities';
+import babel from '@babel/core';
+import { safeLoad } from 'js-yaml';
+import camelCase from 'lodash/camelCase.js';
```

Comment:
is there a good reason for these extensions? seems weird

Reply:
yes they are required for esm imports

Reply:
well not required for packages if the file is `"main"` or `"exports"` in the package.json sets it up. lodash doesn't set anything up